packages/features/shell/Shell.tsx

```diff
@@ -747,9 +747,7 @@ function SideBar() {
         </div>
 
         <div>
-          <LicenseRequired toHide>
-            <Tips />
-          </LicenseRequired>
+          <Tips />
```

Comment:
Do we want these tips for self hosted users?

Reply:
i think so, yea

Reply:
they can remove it if needed